src/interface.rs

```diff
@@ -360,4 +361,38 @@ impl HWIClient {
             status.into()
         })
     }
+
+    /// Get installed version of hwilib. Returns None if not found, or if version couldn't be known
```

Comment:
nit: Get **the** installed version of hwlib. Returns None if **hwi is not installed**.

Reply:
It should be "Get **the** installed version"